### PR TITLE
added response with subscription id, updated delete subscriptions add…

### PR DESCRIPTION
**Pull request by natashareis**
image

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -41,18 +41,28 @@ We expect you to send this information in a field called 'X-Token' inside header
...
     "metadata": "user_id_in_your_system"
 }
+
+//RESPONSE EXAMPLE
+
+{
+    "id": 1,
+    "metadata": "user_1",
+    "delivery_url": "https://api.your_url.com/nestready/events",
+    "events": ['listing_favorited'],
```
Comment (AlexVKO) [inline source/index.html.md]: let's use " instead of '

Comment (natashareis) [reply source/index.html.md]: Done

Comment (natashareis) [reply source/index.html.md]: image

**Comment by natashareis** [comment]
This pull request closes https://wenavut.atlassian.net/browse/ENG-137 and https://wenavut.atlassian.net/browse/ENG-138

**Comment by natashareis** [comment]
image
Please review again have add the Add event new_listings_from_saved_search to docs that closes 
https://wenavut.atlassian.net/browse/ENG-144

**Pull request merged**